apps/condo/pages/tls.tsx

```diff
@@ -0,0 +1,296 @@
+// @ts-ignore
+import enMacos from '@condo/lang/en/pages/tls/macos.md'
+// @ts-ignore
+import enWindows from '@condo/lang/en/pages/tls/windows.md'
+// @ts-ignore
+import ruMacos from '@condo/lang/ru/pages/tls/macos.md'
+// @ts-ignore
+import ruWindows from '@condo/lang/ru/pages/tls/windows.md'
+import styled from '@emotion/styled'
```

Comment:
We're trying not to use emotion in our new code...Can you rewrite it to avoid it?

Reply:
This page have parts with look, used first time in our project.
See link https://www.figma.com/file/GsLjSzbZcEw4Ll82sN3LU3/01-%D0%A0%D0%B5%D0%B3.%2F%D0%90%D0%B2%D1%82%D0%BE%D1%80%D0%B8%D0%B7.?type=design&node-id=1157-8072&mode=design&t=Lwk9zd4pPs66IaW0-4

In my case of rendering markdown, components from '@open-condo' have different look with default styles, that look buggy, like markers of lists.

All the CSS tweaks you see here are styling to look like in Figma design.